src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetFileEncodingCommand.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Management.Automation;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements the Get-FileEncoding command.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "FileEncoding", DefaultParameterSetName = PathParameterSet, HelpUri = "https://go.microsoft.com/fwlink/?LinkID=1234567")]
```

Comment:
We need new link.

Reply:
Yeah put this one as a placeholder. I will also fill out documentation issue.